src/simulator/src/app.js

```
@@ -1,210 +1,58 @@
-import { setup } from './setup'
+import { setup } from './setup';
+import Array from './arrayHelpers';
```

Biome (2.1.2)

[error] Do not shadow the global "Array" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)